bin/pulsar

```diff
@@ -48,6 +52,8 @@ PRESTO_HOME=${PULSAR_HOME}/lib/presto
 # Check bookkeeper env and load bkenv.sh
 if [ -f "$PULSAR_HOME/conf/bkenv.sh" ]
 then
+    BOOKIE_LOG_DIR=${BOOKIE_LOG_DIR:-"${PULSAR_LOG_DIR}"}
```

Comment:
In pulsar-daemon.sh BOOKIE_LOG_DIR  is unified to `PULSAR_LOG_DIR=${BOOKIE_LOG_DIR:-"$PULSAR_HOME/logs"}` . Here, BOOKIE_LOG_DIR is useless.

Reply:
Addressed in latest commit.